r/fratPhyloseq.Rmd

```diff
@@ -0,0 +1,63 @@
+---
+title: "Phyloseqify the Fratjuice samples"
+author: "Richard Sprague"
+date: "1/5/2018"
+output: github_document
+---
+
+Load the prerequisites:
+
+```{r}
+library(phyloseq)
+library(actino)
+library(tidyverse)
+SAMPLEDIR <- "../samples/ubiome/ubiome-processed-outputs"
+
+```
+
+Read the mapping data into a more familiar "Actino-style" mapfile
+
+```{r}
+results <- readxl::read_excel(paste0(SAMPLEDIR,"/results.xlsx"))
```

Comment:
Let's read/generate the sample mapping dataframe from [`samples/ubiome/kits.tsv`](https://github.com/dhimmel/fratjuice/blob/master/samples/ubiome/kits.tsv) rather than `results.xlsx`. Trying to keep the authoritative sample mapping in one and only one place.

Delete `samples/ubiome/ubiome-processed-outputs/json/fratmap.xlsx`.

Reply:
the mapfile needs the SSR.  Also ```samples.tsv``` doesn't contain all samples. Do you want to add the extra columns?

Reply:
Use `kits_id` rather than `samples.tsv`. This should contain everything. The `seq_id` column is the SSR.